js/index.js

```diff
@@ -40,3 +40,59 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+///nAV
+let navAList = document.querySelectorAll("header nav a");
+navAList[0].innerText = siteContent["nav"]["nav-item-1"];
+navAList[1].innerText = siteContent["nav"]["nav-item-2"];
+navAList[3].innerText = siteContent["nav"]["nav-item-4"];
+navAList[4].innerText = siteContent["nav"]["nav-item-5"];
+navAList[5].innerText = siteContent["nav"]["nav-item-6"];
+///END NAV
```

Comment:
Here is an example of doing it the way we talked about in stand up using the forEach method. This code loops through the nodelist you have selected and assigns values from the siteContent object.

```
nav.forEach(function(value, i){
  value.textContent=siteContent.nav[`nav-item-${i+1}`];
});
```